packages/azure-themes/src/stories/components/commandBar.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import { CommandBar, ICommandBarItemProps } from 'office-ui-fabric-react/lib/CommandBar';
+import { IButtonProps } from 'office-ui-fabric-react/lib/Button';
+
+const overflowProps: IButtonProps = { ariaLabel: 'More commands' };
+
+export const CommandBarBasicExample: React.FunctionComponent = () => {
+  return (
+    <div>
+      <CommandBar
+        items={_items}
+        overflowItems={_overflowItems}
+        overflowButtonProps={overflowProps}
+        farItems={_farItems}
+        ariaLabel="Use left and right arrow keys to navigate between commands"
+      />
+    </div>
+  );
+};
+
+const _items: ICommandBarItemProps[] = [
+  {
+    key: 'newItem',
+    text: 'Add',
+    cacheKey: 'myCacheKey', // changing this key will invalidate this item's cache
+    iconProps: { iconName: 'Add' },
+    // subMenuProps: {
+    //   items: [
+    //     {
+    //       key: 'emailMessage',
+    //       text: 'Email message',
```

Comment:
remove commented code?

Reply:
Resolving these comments in upcoming PR.